.github/workflows/openapi-generator-gradle-build.yaml

```diff
@@ -0,0 +1,30 @@
+name: OpenAPI Generator (Gradle Build)
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: temurin
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Build with Gradle
+        run: ./gradlew clean build publishToMavenLocal
+
+      - name: Verify Gradle plugin sample
+        working-directory: modules/openapi-generator-gradle-plugin/samples/local-spec
+        run: |
+          ./gradlew clean buildGoSdk
+          ./gradlew clean openApiGenerate
+          ./gradlew clean buildDotnetSdk
+          ./gradlew clean buildJavaRestTemplateSdk
```

Comment:
Gradle plugin sample verification has been moved to this file so that it may test the artifact produced by this build.